### ISSUE TITLE
slashdot.org: ads / breakage / detection

### ISSUE THREAD

**Issue by hyler** [uBlockOrigin/uAssets#26275]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.). Comment in the old issue threads even when they are closed.
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://slashdot.org/`

### Category

ads

### Description

I used to see absolutely no ads on Slashdot, but lately they have come up in full force. This is Firefox on Windows 10. For the record, in iOS, where I use 1Blocker (with Safari), I've also started seeing them. So I wonder if Slashdot has changed how they serve them, so they can evade the filters. (Creating a custom filter [by using the picker on the elements] completely destroys the whole site.)

### Other extensions used

Privacy Badger (but turning it off does not affect the issue). 

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/user-attachments/assets/9666cabd-74c9-41e6-99ab-6288a77c8aeb)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Firefox: 133
filterset (summary):
 network: 136118
 cosmetic: 48977
 scriptlet: 20727
 html: 1962
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 40154-138, 19m Δ
  ublock-badware: 11105-6, 19m Δ
  ublock-privacy: 1097-2, 19m Δ
  ublock-unbreak: 2528-20, 19m Δ
  ublock-quick-fixes: 85-4, 19m Δ
  easylist: 77737-589, 19m
  easyprivacy: 53053-25, 19m Δ
  urlhaus-1: 20025-0, 19m
  plowe-0: 3540-615, 18m
filterset (user): [empty]
trustedset:
 added: [array of 7 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 661 ms
 maxAssetCacheWait: 239 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 15
 network:
  slashdot.org: 3
  6sc.co: 1
  ad-delivery.net: 2
  btloader.com: 1
  criteo.net: 1
  d3tglifpd8whs6.cloudfront.net: 1
  fsdn.com: 2
  googlesyndication.com: 1
  ml314.com: 1
  slashdotmedia.com: 2
 extended:
  ###announcement
  ###main-nav-badge-link
  ###slashdot_deals
  ##.deals-wrapper
  ##[id^="div-gpt-ad"]:not([style^="width: 1px; height: 1px; posit…
  ##div[id^="div-gpt-"]:not([style^="width: 1px; height: 1px; posi…
  ##.adwrap
  ###slashboxes > .deals-rail
  ##.scroll-fixable.rail-right > .deals-rail
  ##.railad
  #@#[id^="div-gpt-ad"]
  #@#div[id^="div-gpt-"]
```
</details>

**Comment by stephenhawk8054** [comment]
Test again in a new Firefox profile with only uBO and its default settings.

**Comment by hyler** [comment]
Thank you for replying. I see it's been tagged with "unable to reproduce" but I am still seeing the same, even with a new profile and only uBO installed.

![image](https://github.com/user-attachments/assets/278e51bd-a060-4f91-aee0-7f09b6beac72)

**Comment by stephenhawk8054** [comment]
Which country are you when accessing the site?

**Comment by krystian3w** [comment]
Try update add-on to 1.60+

https://addons.mozilla.org/en-us/firefox/addon/ublock-origin/versions/

**Comment by hyler** [comment]
> Try update add-on to 1.60+
> 
> https://addons.mozilla.org/en-us/firefox/addon/ublock-origin/versions/

Looks like I'm on 1.61.2 already.
![image](https://github.com/user-attachments/assets/d34c7a5c-4a3c-4f52-8179-d99d4c475f78)

**Comment by krystian3w** [comment]
Strange, we have HTML filtering to disable Ad-Shield. Try found in logger these:

![obraz](https://github.com/user-attachments/assets/685f4fd4-ddc9-42cc-8417-42b6c7edd706)

**Comment by stephenhawk8054** [comment]
Yeah, I can't reproduce the ads either

![image](https://github.com/user-attachments/assets/69e318d5-67cf-4cf1-9bf7-fd024239b5ff)

**Comment by stephenhawk8054** [comment]
@JobcenterTycoon I think `replace` filters need to be updated?

**Comment by OneOfMany07** [comment]
(I plan to look at other bugs too, but thought I'd post here first)

Only just happened for first time for me (month later plus), but I also get a warning dialog that an ad blocker is causing problems (with a way to get more info).  Wonder if someone is creating malicious ads, but not serving them often or far?  I've got old hardware so that might shift their results slightly.

Guessing the default filter list is safe, but I enabled everything minus all privacy.  Guessing original report used an additional filter list (beyond defaults) until they uninstalled, then were at defaults again.

FF user, in US (haven't moved at all, and don't live near where the other person does...  I'm west coast), using currently released versions of everything.  Including being the 1.61.2 version of ublock origin extension mentioned above.

**Comment by stephenhawk8054** [comment]
Force update the filter list by [clicking on this link](https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-quick-fixes&manual=250116).

**Comment by D3SL** [comment]
> Force update the filter list by [clicking on this link](https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-quick-fixes&manual=250116).

Doesn't help. Slashdot's got the "html-load" malware running that detects adblock and breaks the site.

**Comment by stephenhawk8054** [comment]
They regularly interfere with our filters. If you don't want to wait for fixes, you can use these as your custom filters: https://github.com/stephenhawk8054/css#filters

**Comment by JobcenterTycoon** [comment]
I updated the firefox filter 50 minutes ago but i don’t get Ad-Shield on this site at all so i can’t test.

**Comment by JobcenterTycoon** [comment]
Ok im able to reproduce with a US VPN but with the latest filter the site works fine for me.

**Comment by hyler** [comment]
Interesting that someone reopened this, I already felt so bad that I opened it initially for what seemed a user error (or at least a fluke). The only thing I want to add is that while the issue is "effectively" gone, the way it's "gone" is that sometimes on first load the website will show ads, but then if I refresh the page they'll disappear. It's basically, as the scientists call it, "wonky." Anyway, I'm always willing to run some software or other on my machine if anyone needs me to install something special for diagnosis. Peace.

**Comment by stephenhawk8054** [comment]
Which browser are you using?

**Comment by hyler** [comment]
> Which browser are you using?

Firefox. Have to say, lately I've not seen the issue, intermittent or otherwise.

**Comment by Yuki2718** [comment]
>  Hmm... Looks like exceptions override `replace`?

https://github.com/uBlockOrigin/uAssets/discussions/27216#discussioncomment-12223320

**Comment by stephenhawk8054** [comment]
Yeah, I saw that thread, but actually exceptions did not override `replace` in this case when I checked again. Not sure what happened.

**Comment by ziyaadm** [comment]
Can this be reopened? Running uBlock Origin in Complete mode on Safari.

**Comment by stephenhawk8054** [comment]
Please copy your uBOL troubleshooting information to here and screenshot what issue you are having. Don't just describe too vaguely like that.

**Comment by craigrileyuk** [comment]
It's detecting uBlock when you enter an article page. Sponsored posts are showing in the index, advert is showing in the header.

Additionally, seem to be having trouble loading the browser devtools. They show an empty panel.

```
uBlock Origin: 1.65.0
Firefox: 142
filterset (summary):
 network: 0
 cosmetic: 0
 scriptlet: 0
 html: 0
listset (total-discarded, last-updated):
 removed:
  user-filters: null
  ublock-filters: null
  ublock-badware: null
  ublock-privacy: null
  ublock-unbreak: null
  ublock-quick-fixes: null
  easylist: null
  easyprivacy: null
  urlhaus-1: null
  plowe-0: null
filterset (user): [empty]
trustedset:
 removed:
  chrome-extension-scheme
  moz-extension-scheme
switchRuleset:
 removed:
  no-large-media: behind-the-scene false
  no-csp-reports: * true
hostRuleset:
 removed:
  behind-the-scene * * noop
  behind-the-scene * image noop
  behind-the-scene * 3p noop
  behind-the-scene * inline-script noop
  behind-the-scene * 1p-script noop
  behind-the-scene * 3p-script noop
  behind-the-scene * 3p-frame noop
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 440 ms
 maxAssetCacheWait: 4 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 0
```

**Comment by stephenhawk8054** [comment]
Your uBO is broken. Uninstall and re-install it.

**Comment by craigrileyuk** [comment]
Yeah, for some reason it seems that updating to Firefox 142 broke all of my extensions and devtools as well.
Had to remove Firefox, reinstall and create a new profile to get everything working again.
